uint8 agent_i
uint8 agent_j
geometry_msgs/Quaternion C_ij_rel
